setup.cfg

```
@@ -29,6 +29,7 @@ project_urls =
     Documentation = https://celery-batches.readthedocs.io
     Release notes = https://github.com/clokep/celery-batches/blob/main/CHANGELOG.rst
     Source = https://github.com/clokep/celery-batches
+    Sponsor = https://github.com/sponsors/clokep
```

I think this is supposed to be Funding?

```suggestion
    Funding = https://github.com/sponsors/clokep
```

---

Squashed